modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/DeployApp.es.js

```diff
@@ -20,22 +21,22 @@ import WidgetSettings from './settings/WidgetSettings.es';
 
 const Divider = () => {
 	return (
-		<div className="autofit-row mb-4 pl-2 pr-2">
-			<div className="col-md-12">
+		<ClayLayout.ContentRow className="mb-4 pl-2 pr-2">
+			<ClayLayout.Col md>
```

Comment:
@marcoscv-work do we need the `md`?

Reply:
Probably better a `<ClayLayout.ContentCol>`